docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
+
+It creates and plots a map showing the coriolis parameters, overlaid on top of
+the UK coastline. The code also creates a custom cube to store the data in and
+to plot it from.
+
+"""
+
+import cartopy.crs as ccrs
+
+import iris
+
+from iris.coord_systems import GeogCS
+
+import iris.quickplot as qplt
+
+import matplotlib.pyplot as plt
+
+import numpy as np
+
+
+def calculate_coriolis_parameter(latitude):
+    """
+    This function computes the coriolis parameters for the points in an array.
+
+    It takes a 2D latitude array as its input.
+
+    It returns a 2D array for coriolis parameters.
+    """
+    omega = 7.29E-5
+    return 2. * omega * np.sin(latitude)
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 50
+    latitudes = np.linspace(49.0, 60.0, resolution)
+    longitudes = np.linspace(-11.0, 3.0, resolution)
+    lat_calc = np.deg2rad(latitudes)
+    lon_calc = np.deg2rad(longitudes)
+    lat1, long1 = np.meshgrid(lat_calc, lon_calc)
```

Comment:
The lat and lon are in the opposite order for normal conventions, I'd suggest switching them round then making the cube dimension order latitude longitude.

Also, the names don't make sense, you might consider `lat2d` instead of `lat1` since that nicely describes the content of the variable. Also I wouldn't bother assigning the longitudes to the variable `long1` since they aren't ever used. 

I'd also be tempted not to do the conversion to radians first and include that in the computation of the coriolis (which I think could be inline):

```python
_, lat2d = np.meshgrid(longitudes, latitudes)

omega = 7.29e-5
data = 2. * omega * np.sin(np.deg2rad(lat2d))
```